www/components/itinerum_tutorial/Tutorial.jsx

```diff
@@ -1,11 +1,28 @@
 /*
  * src/app/help-panel/Tutorial.jsx
  */
-
 class TutorialComponent extends React.Component {
+    propTypes: {
+        test: React.PropTypes.string,
+        isValid: React.PropTypes.bool,
+        schema: React.PropTypes.object,
+        result: React.PropTypes.object
+    }
     render() {
+        bool = this.props.isValid;
+        if(bool) {
+            validity = <p>Valid</p>
+        } else {
+            validity = <p>Not Valid</p>
+        }
         return(
                 <div className="content-wrapper">
+                    <div>
+                        {validity}
+                        This is the test -> {this.props.test}
```

Comment:
Big Issue: this.props.test does not return the string I have set it to. I am missing a link between the two files.

Reply:
Fixed! Needed to include the vars into the <tutorial/> call, much like is done within the Hello.jsx example. 
`<tutorial test="surveyState.test" isValid="surveyState.isValid" schema="surveyState.schema" result="surveyState.result"/>`